DQM/SiPixelCommon/python/SiPixelOfflineDQM_source_cff.py

```diff
@@ -31,6 +31,10 @@
 from DQM.SiPixelMonitorTrack.SiPixelMonitorEfficiency_cfi import *
 SiPixelHitEfficiencySource.saveFile = False
 SiPixelHitEfficiencySource.trajectoryInput = cms.InputTag('generalTracks') 
+from DQM.TrackerMonitorTrack.SiPixelMonitorTrackResiduals_cfi import *
+SiPixelMonitorTrackResiduals.trajectoryInput = 'generalTracks'
+SiPixelMonitorTrackResiduals.Tracks          = 'generalTracks'
+SiPixelMonitorTrackResiduals.Mod_On          = False
```

Comment:
@davidlange6 what should the configuration in the _cfi do by default? As far as I see all usages of the old `MonitorTrackResiduals` override the default, so I don't know which scenario the default was good for...

Maybe the _cfi's should be changed to `generalTracks` (which seems to be most popular). Note also that some configs set `trajectoryInput` and `Tracks` to different values, which as far as I see, never made sense.